internal/cli/cmd/auth/githubtoken.go

```diff
@@ -49,7 +49,11 @@ func NewExchangeGithubTokenCmd() *cobra.Command {
 				tok = fmt.Sprintf("nsct_%s", res.TenantToken)
 			}
 
-			return auth.StoreTenantToken(tok)
+			if _, err := auth.StoreUser(ctx, &auth.UserAuth{Username: auth.GithubJWTAudience}); err != nil {
```

Comment:
I think we should not store fake users. This will easily confuse us later.
In case of a github workflow, there is no user, but we can still generate tokens.
Can we cache them with an empty user name?

Reply:
Done, doing cache with empty user.